src/internal/views/nunjucks/billing/two-part-tariff-review.njk

```diff
@@ -0,0 +1,89 @@
+{% extends "./nunjucks/layout.njk" %}
+{% from "badge.njk" import badge %}
+{% from "paginate.njk" import paginate %}
+{% from "./nunjucks/billing/macros/batch-metadata.njk" import batchMetadata %}
+{% from "govuk/components/button/macro.njk" import govukButton %}
+{% from "./nunjucks/billing/macros/totals.njk" import chargeOrDash %}
+
+{% block content %}
+<div class="govuk-grid-row govuk-!-margin-bottom-0">
+  <div class="govuk-grid-column-two-thirds">
+    <section class="govuk-!-margin-bottom-6">
+      {% set caption %}
+        {{ 'Bill run ' + batch.externalId if batch.externalId else null }}
+      {% endset %}
+        {{ title(pageTitle, 'This is the caption', null, true, true) }}
```

Comment:
Should this say 'This is the caption' or use the caption variable defined above?

Reply:
done